src/theme/src/hooks/usePaneAppearance.js

```diff
@@ -0,0 +1,17 @@
+import useStyleConfig from '../../../hooks/use-style-config'
+import getPaneStyles from '../components/pane'
+import useTheme from '../useTheme'
+
+const pseudoSelectors = {
+  _hover: '&:hover',
+  _active: '&:active'
+}
+
+function usePaneAppearance(props, internalStyles) {
```

Comment:
`internalStyles` is nonexistent at the callsite here, can we just add an empty obj for now as a default arg?

Reply:
That happens internally already, so we're good. I'd rather keep the useXAppearance signature consistent throughout